lib/util/status_card.dart

```diff
@@ -54,11 +54,12 @@ class StatusCardState extends State<StatusCard> {
     // When paused, show current print progress.
     // When canceled, show a full circle.
     final circleProgress =
-        (widget.isPausing && widget.status['paused'] != true) || (widget.isCanceling && widget.status['layer'] != null)
+        (widget.isPausing && widget.status['paused'] != true) ||
+                (widget.isCanceling && widget.status['layer'] != null)
             ? null
             : widget.progress == 0.0
                 ? 1.0
-                : widget.progress;
+                : 1.0;
```

Comment:
Should this be simplified to remove the second ternary statement?

Reply:
Resolved by [34f8b4b](https://github.com/TheContrappostoShop/Orion/pull/14/commits/34f8b4bbb34367a14d65e2d1ba2c5495ad53d45e)